src/foam/nanos/boot/NSpec.js

```diff
@@ -278,7 +278,7 @@ foam.CLASS({
       code: function() {
         var service = this.__context__[this.name];
         if ( foam.dao.DAO.isInstance(service) ) {
-          this.ctrl.memento.value = 'admin.data' + foam.nanos.controller.Memento.SEPARATOR + this.name;
+          this.pushMenu(foam.nanos.menu.Menu.create({ id: 'admin.data' + foam.nanos.controller.Memento.SEPARATOR + this.name }));
```

Comment:
hmm the applicationController.pushMenu - checks if the menu through the menuDAO to confirm you have permission to that menu - so creating a menu here isn't going to work.
However applicationController.pushMenu will consider the memento.value being updated ... so don't think this change will be beneficial.

Reply:
Does the aplicationController.pushMenu listen to memento changes?

Reply:
Updated